### ISSUE TITLE
How to uninstall SELinux?

### ISSUE THREAD

**Issue by pablogzlezmora** [archlinuxhardened/selinux#94]
Please, How can I uninstall SELinux?

System: Arch Linux LTS

**Comment by fishilico** [comment]
Hello,
I have never uninstalled SELinux, but if I were to do this, I would perform the following steps:

1. Disable SELinux in the kernel boot command line (for example by adding `selinux=0`)
2. Reboot
3. Ensure that SELinux is disabled by running `sestatus`. This should display `SELinux status: disabled`
4. Replace `...-selinux` packages with their usual ones (for example `systemd-selinux` -> `systemd`), using `pacman -S ...`
5. Reboot, so that core system components (such as systemd, pam, sudo, coreutils...) no longer depends on `libselinux`.
6. Uninstall packages from the group `selinux` (the list can be found by running `pacman -Qqg selinux`)

**Comment by pablogzlezmora** [comment]
@fishilico: It worked.

In step 4, I tried:
`# pacman -S sudo dbus cronie shadow util-linux systemd-resolvconf systemd-sysvcompat systemd pambase pam coreutils`

Between steps 5 and 6, I had to add my user to the _sudoers_ group again (now, I know that it's an obviously needed task).

In step 6, I saw that I had forgotten some packages when I made step 4: `pacman -Qi <package>`, the wiki, and a small amount of patience were very useful here.

The setting seems to be ready now.

Thanks a lot.
